transport/src/main/java/io/netty/channel/EventLoop.java

```diff
@@ -27,4 +27,8 @@
 public interface EventLoop extends EventExecutor, EventLoopGroup {
     @Override
     EventLoopGroup parent();
+
+    void onEventLoopIteration(Runnable task);
+
+    boolean removeOnEventLoopIterationTask(Runnable task);
```

Comment:
same comment as for `ChannelConfig`... we can not add methods here yet. That said I also think we should just add these to `SingleThreadEventExecutor` and do a cast if the `Channel` uses a sub-class. If not we can just not support the new AUTO_FLUSH thingy.